backend/src/ts/auth/check.ts

```diff
@@ -0,0 +1,43 @@
+import * as express from "express";
+import { getCollection } from "../db/collections/users";
+
+export async function isAuthed(
+  req: express.Request,
+  res: express.Response,
+  next: any
+) {
+  if (
+    !req.header("Authorization") ||
+    !req.header("Authorization").startsWith("Bearer")
+  ) {
+    return res
+      .status(400)
+      .send({ error: "Invalid Authorization methiod.", status: 400 });
+  }
+
+  const token = req.header("Authorization").replace("Bearer ", "");
+
+  const user = await getCollection()
+    .findOne({ token })
+    .catch((e: any) => {
+      res
+        .status(500)
+        .send({
+          error: "There was a interal server error, please try again.",
```

Comment:
Add "If this keeps happening, please contact the developers." To this and others (similar things too)